packages/text-input/src/TextInput.tsx

```diff
@@ -323,9 +323,9 @@ const TextInput: React.ComponentType<
       }
     }
 
-    if (!label && !ariaLabelledBy) {
+    if (!label && !ariaLabelledby) {
       console.error(
-        'For screen-reader accessibility, label or aria-labelledby must be provided to TextInput.',
+        'For screen-reader accessibility, label or aria-labelledby must be provided to TextArea.',
```

Comment:
Looks like this is the wrong component name.

Reply:
ty